test-plans/ehrbase_catalonia_rfp_queries.jmx

```diff
@@ -0,0 +1,331 @@
+}</stringProp>
+                <stringProp name="Argument.metadata">=</stringProp>
+              </elementProp>
+            </collectionProp>
+          </elementProp>
+          <stringProp name="HTTPSampler.domain"></stringProp>
+          <stringProp name="HTTPSampler.port"></stringProp>
+          <stringProp name="HTTPSampler.protocol"></stringProp>
+          <stringProp name="HTTPSampler.contentEncoding"></stringProp>
+          <stringProp name="HTTPSampler.path">/ehrbase/rest/openehr/v1/query/aql</stringProp>
+          <stringProp name="HTTPSampler.method">POST</stringProp>
+          <boolProp name="HTTPSampler.follow_redirects">true</boolProp>
+          <boolProp name="HTTPSampler.auto_redirects">false</boolProp>
+          <boolProp name="HTTPSampler.use_keepalive">true</boolProp>
+          <boolProp name="HTTPSampler.DO_MULTIPART_POST">false</boolProp>
+          <stringProp name="HTTPSampler.embedded_url_re"></stringProp>
+          <stringProp name="HTTPSampler.connect_timeout"></stringProp>
+          <stringProp name="HTTPSampler.response_timeout"></stringProp>
+        </HTTPSamplerProxy>
+        <hashTree>
+          <ResponseAssertion guiclass="AssertionGui" testclass="ResponseAssertion" testname="Response Assertion" enabled="true">
+            <collectionProp name="Asserion.test_strings">
+              <stringProp name="49586">200</stringProp>
+            </collectionProp>
+            <stringProp name="Assertion.custom_message"></stringProp>
+            <stringProp name="Assertion.test_field">Assertion.response_code</stringProp>
+            <boolProp name="Assertion.assume_success">false</boolProp>
+            <intProp name="Assertion.test_type">16</intProp>
+          </ResponseAssertion>
+          <hashTree/>
+        </hashTree>
+        <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="Get all patients which have a specific diagnosis for a specific health care facility" enabled="true">
+          <boolProp name="HTTPSampler.postBodyRaw">true</boolProp>
+          <elementProp name="HTTPsampler.Arguments" elementType="Arguments">
+            <collectionProp name="Arguments.arguments">
+              <elementProp name="" elementType="HTTPArgument">
+                <boolProp name="HTTPArgument.always_encode">false</boolProp>
+                <stringProp name="Argument.value">{&#xd;
+  &quot;q&quot;: &quot;SELECT DISTINCT e/ehr_id/value FROM EHR e CONTAINS COMPOSITION c CONTAINS EVALUATION ev[openEHR-EHR-EVALUATION.problem_diagnosis.v1] WHERE c/context/health_care_facility/name/value=&apos;Hospital&apos; AND ev/data[at0001]/items[at0002]/value/value=&apos;Problem/Diagnosis name 10&apos;&quot;&#xd;
```

Comment:
c/context/health_care_facility/name/value='hcf0hcp0'

instead of 'Hospital'

Reply:
Hello @vmueller-vg,
I updated the query, as it didn't return any results. With 'Hospital' it is returning something.
PFB screenshot:
![image](https://user-images.githubusercontent.com/100927963/177265530-f324341d-7ceb-48a5-9153-a6d83db84027.png)
In below query, I selected all _c/context/health_care_facility/name/value_ and seems that no other values are present, except this 'Hospital'.
Query: 
`SELECT DISTINCT e/ehr_id/value, c/context/health_care_facility/name/value FROM EHR e CONTAINS COMPOSITION c CONTAINS EVALUATION ev[openEHR-EHR-EVALUATION.problem_diagnosis.v1] WHERE ev/data[at0001]/items[at0002]/value/value='Problem/Diagnosis name 10'`
![image](https://user-images.githubusercontent.com/100927963/177266177-9eb869e8-5f1c-4625-a29e-b1dda02054f5.png)